ZenPacks/zenoss/HttpMonitor/http.py

```diff
@@ -128,10 +128,28 @@ def _lookupErr(self, failure):
             )
         return failure
 
+    def _lookupProxyIpErr(self, failure):
+        return Failure(
+            exc_value=RuntimeError("Unable to resolve hostname (%s) as IP address" % (self._ipAddr)),
+            exc_type=RuntimeError
+            )
+
+    def _lookupProxyIp(self, response):
+        if response:
+            for a in response[0]:
+                if a.payload.TYPE == dns.A:
+                    self._ipAddr = a.payload.dottedQuad()
+                    break
+        return self.connect()
+
     def connect(self):
+        if not isip(self._ipAddr):
+            return client.lookupAddress(self._ipAddr).addCallbacks(
+                    self._lookupProxyIp, self._lookupProxyIpErr
```

Comment:
This line has one indent too many.   Please indent only four spaces under the `return`.